test/composite.jl

```diff
@@ -0,0 +1,45 @@
+@testset "composite.jl" begin
+    @testset "constructor" begin
+        id = IdentityScaling()
+        power = Power(3.0)
+        logt = LogTransform()
+        @test id ∘ id == Composite([id, id])
+        @test id ∘ id ∘ power == Composite([power, id, id])
+        @test power ∘ id ∘ power == Composite([power, id, power])
+
+        @test power ∘ (id ∘ logt) == Composite([logt, id, power])
+        @test (power ∘ id) ∘ logt == Composite([logt, id, power])
+        @test (power ∘ id) ∘ (logt ∘ id) == Composite([id, logt, id, power])
+
+        @test_throws ArgumentError id ∘ LinearCombination([1, 2, 3])
+        @test_throws ArgumentError OneHotEncoding([1, 2]) ∘ id
```

Comment:
alternatively we could use `test_skip` until these are implemented?

Reply:
I wanted to make sure there is an actual error, so possibly @test_broken is the right thing